theforeman.org/pipelines/lib/rbenv.groovy

```diff
@@ -1,8 +1,8 @@
-def bundleInstall(version, gemfile=null) {
-    command = "bundle install"
+def bundleInstall(version, options=null) {
+    command = "bundle install --jobs 5 --retry 5"
```

Comment:
`theforeman.org/pipelines/test/foreman-installer.groovy` and `theforeman.org/pipelines/test/simple-ruby.groovy` are using the `gemfile` param and would break with this change.

Reply:
Ahh yea, thanks. Got it.